### DISCUSSION TITLE
How can I access front end?

### DISCUSSION THREAD

**Discussion by 23sergej** [austintoddj/canvas#1393] [General]
I've create a post, but there is no way to see how it looks for end user...

**Comment by austintoddj** [comment]
Hi @23sergej - if you [follow these steps](https://github.com/austintoddj/canvas#canvas-ui), you'll be able to generate a frontend. By default Canvas doesn't ship with one so users can have more flexibility, but if you need to spin something up quickly then those steps are a great way to do it.

**Comment by 23sergej** [comment]
Thank you @austintoddj ;)
